src/components/Work/ParentCollections.js

```diff
@@ -5,42 +5,44 @@ import ThisItem from "./ThisItem";
 import SectionTop from "../UI/SectionTop";
 import { reactiveSearchFacets } from "../../services/reactive-search";
 
-const ParentCollections = ({ adminSetItems = [], collection, item }) => {
-  const adminSetTitle = item.admin_set ? item.admin_set.title[0] : "";
+const ParentCollections = ({ libraryUnitItems = [], collection, item }) => {
+  const libraryDepartment = item.administrativeMetadata.libraryUnit
+    ? item.administrativeMetadata.libraryUnit.label
+    : "";
   return (
     <div>
-      {item && adminSetItems.length > 0 && (
+      {item && libraryUnitItems && libraryUnitItems.length > 0 && (
```

Comment:
Since you're setting a default value above `libraryUnitItems = []`, you can remove `libraryUnitItems` here.